.claude/skills/startproject/SKILL.md

```
@@ -372,6 +382,24 @@ prompt: |
     Report only Critical/High findings.
 ```
 
+**設計書との突合**: `docs/` 配下に承認済みの設計書（`/design` の出力）が存在する場合は、上記に加えて `spec-reviewer` で実装と設計書の整合を確認する:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)